### PR TITLE
Fixes catbans not applying

### PR DESCRIPTION
**Pull request by yoyobatty**
[Guidelines]: # (Be sure that your PR follows our guidelines, such as modularization and comment standards. You can read more about the subject here: https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md )

[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs )

:cl: YoYoBatty
admin: Catbans should change species 
/:cl:

[why]: Boom

### CODE REVIEW

File: hippiestation/code/controllers/subsystem/job.dm [hunk 1]
```diff
@@ -24,4 +24,4 @@
 	for(var/p in unassigned)
 		var/mob/dead/new_player/player = p
 		if(is_banned_from(player.ckey, CLUWNEBAN) || is_banned_from(player.ckey, CATBAN))
-			AssignRole(player, overflow_role)
+			AssignRole(player, SSjob.overflow_role)
```
Comment (Carbonhell) [inline hippiestation/code/controllers/subsystem/job.dm:27]: why is this needed? src is SSjob

Comment (k-k-karma) [reply hippiestation/code/controllers/subsystem/job.dm]: explicit src looks cooler
